example/main.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const view = require('./view');
+
+const Stator = require('../stator');
+
+var element = document.body;
+
+var state = new Stator(element);
+
+state.observe('keyup', ev => ({input: ev.target.value}), function (state, action) {
```

Comment:
@leted here's the major change; there's now one call to Stator.observe() for each event you want to handle.